Contents/Code/__init__.py

```diff
@@ -37,24 +53,19 @@ def search_na(results, media_title, year, lang):
     searchURL = EXC_BASEURL + 'dev/' + query_actor
     Log('Search URL: ' + searchURL)
     search_results = HTML.ElementFromURL(searchURL)
-  
-  xp = '''//select//Option[text()[contains(translate(., "%s", "%s"), "%s")]]//@value''' % (na_url_site.upper(), na_url_site.lower(), na_url_site.lower())
-  Log('xPath: ' + xp)
+
   try:
+    searchURL = find_option_value(searchURL, search_results, search)
```

Comment:
Good catch, it's been working so far without it though, which begs to question as to why, I'll update and look into that